### DISCUSSION TITLE
error command space login cli

### DISCUSSION THREAD

**Discussion by fernandorrn3** [deta/space-docs#89] [Q&A]
my windows is 8.1 , my node version 18.16.0
![image](https://user-images.githubusercontent.com/97811370/233876850-32f1adda-5efc-4f6e-8622-75ca669f1be2.png)

**Comment by sudo-subg** [comment]
hey @fernandorrn3, try using the CLI in [Windows Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701). If that doesn't work try avoiding the interactive mode with something like this `echo "<your-token>" | space login --with-token`.

Btw, support for window 8.1 has ended, you should consider upgrading your windows version.
